ivadomed/postprocessing.py

```diff
@@ -137,3 +138,52 @@ def mask_predictions(predictions, mask_binary):
     assert predictions.shape == mask_binary.shape
     assert np.array_equal(mask_binary, mask_binary.astype(bool))
     return predictions * mask_binary
+
+
+def coordinate_from_heatmap(nifti_image, thresh=0.3, slice_axis=0):
+    """
+    Retrieve coordinate from a soft output in a 2D image
+    Args:
+        nifti_image(nibabel object): Single slice nifti image of the soft output.
+        thresh(float):Relative threshold for local maxima.
+
+    Returns:
+        list: A list of computed coordinates found by local maximum in RAS convention
```

Comment:
why in RAS convention? if user inputs a slice in LPI, i would expect my list of coordinates to be in LPI (otherwise it might break things down the line)